SPECS/docsis.spec

```diff
@@ -1,12 +1,12 @@
 Name: docsis
-Version: 0.9.8.r391.g6b23fd2
-Release: 2
+Version: 0.9.8.r393.gd7c9643
+Release: 1
```

Comment:
why release down to 1 again? 2 makes sense here or not?

Reply:
https://rpm-packaging-guide.github.io/ -> search for "Preamble Items" and in the table for "Release".

Reply:
> Reset to 1 when a new Version of the software is built.

ah okay, will approve then